lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1387,6 +1390,44 @@ SPIRVExtInst *LLVMToSPIRVDbgTran::getSource(const T *DIEntry) {
   return Source;
 }
 
+void LLVMToSPIRVDbgTran::generateBuildIdentifierAndStoragePath(
+    const DICompileUnit *DIEntry) {
+  // get information from LLVM IR
+  auto BuildIdentifier = DIEntry->getDWOId();
+  const std::string StoragePath = DIEntry->getSplitDebugFilename().str();
+
+  if (LLVMToSPIRVDbgTran::BuildIdentifierAndStoragePathGenerated) {
+    // BuildIdentifier and StoragePath instructions already generated
+    assert(LLVMToSPIRVDbgTran::BuildIdentifier == BuildIdentifier &&
+           "New BuildIdentifier should match previous BuildIdentifier");
+    assert(LLVMToSPIRVDbgTran::StoragePath == StoragePath &&
+           "New StoragePath should match previous StoragePath");
+    return;
+  }
+
+  // generate BuildIdentifier insn
+  SPIRVWordVec BuildIdentifierOps(
+      SPIRVDebug::Operand::BuildIdentifier::OperandCount);
+  const std::string IdString = std::to_string(BuildIdentifier);
+  BuildIdentifierOps[SPIRVDebug::Operand::BuildIdentifier::IdentifierIdx] =
```

Comment:
What about adding using namespace SPIRVDebug::Operand and changing ?
```suggestion
  BuildIdentifierOps[BuildIdentifier::IdentifierIdx] =
```

Reply:
Updated.